RickyChien.md

```diff
@@ -1,10 +1,33 @@
-##  ~ 08/12
+## 08/15 - 08/19
 
-[Control Center]
+###Control Center
 
-* [Fixed] [Bug 1291998](http://bugzil.la/1291998) - Background color of cancel button in permission dropdown on Windows is distinct from OS X and Linux
-* [r?] [Bug 1206229](http://bugzil.la/1206229) - Inform the user that changes may require a page reload
+####Fixed
```

Comment:
This is not the right format. I need the status to be under every bug (not the other way around).